### ISSUE TITLE
website content

### ISSUE THREAD

**Issue by tbreloff** [JuliaML/JuliaML.github.io#1]
I think it would be a really great first step to create a summary of the current state of machine learning in Julia here.  We should show a list of relevant packages along with some high-level information about each one.  It would also be great to have a few small examples/tutorials for "hello world" model fittings in some of the core packages.
### Packages to put on the website (in no particular order):
- Mocha.jl
- MXNet.jl
- OnlineAI.jl
- KSVM.jl
- Clustering.jl
- DecisionTrees.jl
- Orchestra.jl
- LearnBase.jl
- MLBase.jl
- MLKernels.jl
- MachineLearning.jl
- https://github.com/dmlc/XGBoost.jl

What am I missing?

We can also link to https://github.com/svaksha/Julia.jl/blob/master/AI.md for anything we don't highlight.

---

Now would also be a good time for me to start work on MLPlots.jl (probably instead of StatPlots.jl, since [no one showed any interest in that](https://github.com/JuliaStats/Roadmap.jl/issues/18), and then I can combine things like ROC curves, etc).  Then we could provide some common visualizations on the website as well.

**Comment by tbreloff** [comment]
I started us off with a base website, using JuliaStats as a rough template.  Content needed!

**Comment by tbreloff** [comment]
Can we agree on the packages we'd like to highlight here?  I'm going to start a list in the first comment... please post anything that is missing.  Note: we don't need to actually move these packages into the org... I just want to have them on the website (with links to the github pages) so it's all consolidated.

**Comment by Evizero** [comment]
> Now would also be a good time for me to start work on MLPlots.jl (probably instead of StatPlots.jl, since no one showed any interest in that, and then I can combine things like ROC curves, etc)

I think such a package would be very important at some point. I am a little reserved at the moment because I don't want to dillude my attention from KSVM too much (I am already kinda on a tangent with the loss functions in LearnBase) after that I would most certainly contribute the SVM plots to it.

ps: It's not set in stone yet, but it looks like I will give an hour long "Introduction to ML" talk at the end of this month in which I will use Julia examples to explain basic concepts such as linear/logistic regression, penalties, loss functions, gradient descent and such. It might be the case that I'll have to implement some common plots for that, which I would gladly contribute

**Comment by tbreloff** [comment]
Cool... feel free to use anything from ExamplePlots... I presented at the last julia meetup in NYC and I have lots of stuff [here](https://github.com/tbreloff/ExamplePlots.jl/tree/master/examples/meetup)... some of it might be useful.

**Comment by tqchen** [comment]
https://github.com/antinucleon/XGBoost.jl

**Comment by antinucleon** [comment]
@tqchen 
Now it is part of DMLC: https://github.com/dmlc/XGBoost.jl

**Comment by Evizero** [comment]
@tbreloff could you go ahead and establish MLPlots.jl? I would kick it off by creating plots for the loss functions. I think it should live at your repo until it's mature enough to move to JuliaML.

**Comment by tbreloff** [comment]
Working on this now.  Initial commit is up... I'm working to make sure the `corrplot` works with the example in the readme.

https://github.com/tbreloff/MLPlots.jl

**Comment by tbreloff** [comment]
Ok it's ready for contributions!
